Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -21,6 +21,10 @@ open class ChatMessageListVC: _ViewController,
     UIGestureRecognizerDelegate,
     VoiceRecordingAttachmentPresentationViewDelegate
 {
+    open var dataSourceDecorator: (([ChatMessage]) -> [ChatMessage])? = {
```

Comment:
shouldn't we return `nil` as default? This will filter system messages in the SDK.

Reply:
Yes, yes, I'm just testing this to see if Firebase Performance Metrics has any impact or not